apps/medon-fe/src/components/Navigation/index.tsx

```diff
@@ -21,6 +29,13 @@ import { navigation } from 'utils/constants/navigation';
 export default function Navigation() {
   const { t } = useTranslation();
 
+  const user = useAppSelector(getUserSelector);
+  const dispatch = useDispatch();
+  const logout = () => {
+    dispatch(setUser(initialState.user));
+    dispatch(setToken(''));
```

Comment:
change for logout reducer, cause this 2 doesn't clean all userState

Reply:
done